### PR TITLE
Milestone 2 - React Calculator App

### PR DESCRIPTION
**Pull request by JohnHernCode**
Milestones met:
1. Prepare the directory structure
2. Create a components directory in the src folder.
3. Create four files in there. 
4. Implement the App component
5. App renders the Display and ButtonPanel component. 
6. Wrap the children components with React.Fragment.
7. Implement the Display component (default prop is 0)
8. Implement the Button component (accepts button name as prop [type is string])
9. Implement the ButtonPanel component (renders the calculator panel with buttons in groups)

Started thinking about the functionality, but haven't implemented anything concrete, just a bunch of "going to use" code. 

Result: a pretty ugly page full of non-functional buttons. Don’t worry, you will add some functionality in the next milestone!

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+/* eslint-disable react/no-access-state-in-setstate */
```
Comment (rukundoeric) [inline src/components/App.js]: - [x] [MANDATORY] Kindly avoid the inline ESLint disabling rule, this is equally the same as altering or changing our linter files. Let us fix it

File: src/components/Display.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+/* eslint-disable react/prop-types */
```
Comment (rukundoeric) [inline src/components/Display.js]: - [x] [MANDATORY] Kindly avoid the inline ESLint disabling rule, this is equally the same as altering or changing our linter files. Let us fix it

**Review by rukundoeric — CHANGES_REQUESTED** [review]
# Status: ♻️ CHANGES REQUIRED ♻️
_Great job guys so far 👏 💯 👍. However, there are few issues we need to address before we move on, please check the comments below._

- ### To Highlight
  - Follows gitwork-flow.  💯
  - Codebase looks clean and professional. 💯 


- ### To Improve
   - [x] [MANDATORY] `src/logo.svg`: this image is not used anywhere, kindly remove it.
   - [ ] Check the comments below ...

_Please make all the required changes before you submit another code review. If you have any problems or concerns, you can reach out to me on Slack: Rukundo Eric 🚀 . I wish you all the very best 🥇_

### _Keep working hard._
### _Happy Coding😃_
-----------------------------------------------------------------------------------------------------------------------------------------
**Regards,
[Rukundo Eric](https://rukundo-eric.herokuapp.com/) ,
[LinkedIn](https://www.linkedin.com/in/rukundo-eric/), 
Slack: Rukundo Eric**

File: src/components/Display.js (hunk 2 above)
Comment (JohnHernCode) [reply src/components/Display.js]: Hehehehe this one took me a while to figure out why it wasn't working... lol..

**Review by oloomoses — APPROVED** [review]
## Congratulations, Milestone Approved. :muscle: :tada: :clap: 
**Hello @JohnHernCode,
Great job working on the suggested changes from the previous review.
You have passed all the requirements for this milestone :muscle:**

**Feel free to merge and close this pull request.**

#### Happy Coding
_Slack: Oloo Moses_

**Pull request merged**